### DISCUSSION TITLE
snmp_exporter monitoring a Huawei switch: the manufacture-date timestamp is queryable in Nightingale but not in Grafana - why?

### DISCUSSION THREAD

**Discussion by harvey945** [ccfos/nightingale#1159] [Q&A]
![image](https://user-images.githubusercontent.com/93895686/190939277-d60d4855-0ef5-49de-a19d-c7cf55d5be65.png)

![image](https://user-images.githubusercontent.com/93895686/190939390-79bd23ed-3e8e-4dde-b9ed-a16148b60515.png)

![image](https://user-images.githubusercontent.com/93895686/190939366-adee6e5c-3e30-463e-97bd-a3e514a8d7d4.png)

**Comment by UlricQin** [comment]
首先要保证两者是读取的一个数据源，另外可以打开chrome开发者工具，看看分别发的什么请求，对比一下request和response
